From 18f00c25dfe2a05a7f27d9e60a2ed379c01adb01 Mon Sep 17 00:00:00 2001
From: John Stultz <john.stultz@linaro.org>
Date: Thu, 5 Mar 2020 05:58:56 +0000
Subject: [PATCH 673/830] ANDROID: gki_defconfig: Enable CONFIG_REMOTEPROC
X-NVConfidentiality: public

This patch adds required CONFIG_REMOTEPROC to the gki_defconfig
in order for functionality like wifi to work on the db845c.

Bug: 146449535
Change-Id: I21a233b193b22b84ebb1462daf324bcc6b5c6594
Signed-off-by: John Stultz <john.stultz@linaro.org>
---
 arch/arm64/configs/gki_defconfig | 1 +
 arch/x86/configs/gki_defconfig   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index 4bb2165..df2ab96 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -366,6 +366,7 @@ CONFIG_COMMON_CLK_SCPI=y
 CONFIG_HWSPINLOCK=y
 CONFIG_MAILBOX=y
 CONFIG_ARM_SMMU=y
+CONFIG_REMOTEPROC=y
 CONFIG_RPMSG_CHAR=y
 CONFIG_QCOM_COMMAND_DB=y
 CONFIG_DEVFREQ_GOV_PERFORMANCE=y
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index 9d50af11..ef128a8 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -316,6 +316,7 @@ CONFIG_STAGING=y
 CONFIG_ASHMEM=y
 CONFIG_ION=y
 CONFIG_ION_SYSTEM_HEAP=y
+CONFIG_REMOTEPROC=y
 CONFIG_RPMSG_CHAR=y
 CONFIG_PM_DEVFREQ=y
 CONFIG_IIO=y
-- 
2.7.4

